src/main/steps/respondent/hub-page/content.ts

```diff
@@ -51,15 +51,13 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
       to submit the ‘answer a ${isDivorce ? 'divorce' : 'dissolution'}’ form. This is the form for disputing ${
       isDivorce ? 'the divorce' : 'ending your civil partnership'
     }. You can <a class="govuk-link" href="https://www.gov.uk/government/publications/form-d8b-answer-to-a-divorcedissolutionjudicial-separation-or-nullity-petitionapplication">download the form here</a>.`,
-    line3: `Fill in the form and email it to: <a class="govuk-link" href="mailto:${
-      isDivorce ? 'contactdivorce@justice.gov.uk' : 'civilpartnership.case@justice.gov.uk'
-    }">${isDivorce ? 'contactdivorce@justice.gov.uk' : 'civilpartnership.case@justice.gov.uk'}</a>`,
+    line3: `Fill in the form and email it to: <a class="govuk-link" href="mailto:${contactEmail}">${contactEmail}</a>`,
     line4: `<div class="govuk-body">Or post it to:</div>
       Courts and Tribunals Service centre<br>
       HMCTS ${isDivorce ? 'Divorce Service' : 'Ending Civil Partnerships'}<br>
```

Comment:
does these need to be changed?
so no Courts and Tribunals Service centre<br>
and condition

Reply:
Spoke with Sanchita and Linus, additional changes to logic are unknown but for this ticket for now we keep everything as is and simply update the address / emails